heartbeat/proto/heartbeat.proto

```diff
@@ -0,0 +1,24 @@
+syntax = "proto3";
+
+package proto;
+
+option go_package = "heartbeat";
+
+// HeartbeatV2 represents the heartbeat message that is sent between peers from the same shard containing
+// current node status
+message HeartbeatV2 {
+  bytes   Payload         = 1;
+  string  VersionNumber   = 2;
+  string  NodeDisplayName = 3;
+  string  Identity        = 4;
+  uint64  Nonce           = 5;
+}
+
+// PeerAuthentication represents the DTO used to pass peer authentication information such as public key, peer id,
+// payload and the signature. This message is used to link the peerID with the associated public key
+message PeerAuthentication {
+  bytes   Pubkey           = 1;
```

Comment:
Can you also add field: Signature?
there will be 2 signatures, one for the payload which you already added and one for authentication